tests/test_config_generator.py

```diff
@@ -56,7 +56,8 @@ def setUp(self):
         dynamic_config['postgresql']['parameters'] = dict(dynamic_config['postgresql']['parameters'])
         del dynamic_config['standby_cluster']
         dynamic_config['postgresql']['parameters']['wal_keep_segments'] = 8
-        dynamic_config['postgresql']['use_pg_rewind'] = True
+        dynamic_config['postgresql']['use_pg_rewind'] = True \
```

Comment:
Please use the construction similar to https://github.com/zalando/patroni/pull/3063/files#diff-cfee4a0506f1e512ce4c1fb59d7f7bac2f7924ad20e7d6a679044f4c5f4d6c88R271-R272